scripts/select_msoas.py

```diff
@@ -1,12 +1,37 @@
 # This script helps you define new study areas. You can write the output to a
-# new file, config/some_region.csv .
-
+# new file, config/some_region.csv.
 import csv
+import click
+
+@click.command()
+@click.option('--lda',
+             default='e.g Liverpool',
+             prompt = "Name of Local Authority District",
+             help="Enter the name of the Local Authority District of your preference, to get the MSOA codes for that area")
+
+def area(lda):
+    newarea= []
+    # You must run the SPC pipeline to download this file
+    click.echo("Querying MSOA codes for, %s!" % lda)
+    newFileNameLocation = "config/"+lda+".csv"
```

Comment:
The output filenames don't match existing stuff in `config`. We should pick a consistent style. Do you prefer `west_yorkshire`, `WestYorkshire`, `West_Yorkshire`, something else?

Reply:
We had a similar debate in ASPICS, and I think you suggested to agree on lowcase_newareas e.g west_yorkshire. Then I will do it like this.